column/api/backend/__init__.py

```diff
@@ -0,0 +1,6 @@
+import cache
+
+
+def get_api():
```

Comment:
this name is very confusing. Why is this named get_api? get_cache would be better, or even "get_store", since in the end what we need here is some sort of store to keep our state.